guides/api.md

```diff
@@ -241,27 +241,30 @@ the specified number of milliseconds before returning.
 
 ### ao.sampling(item) ⇒ <code>boolean</code>
 Determine if the sample flag is set for the various forms of
-metadata.
+xtrace data.
 
 **Kind**: static method of [<code>ao</code>](#ao)  
 **Returns**: <code>boolean</code> - - true if the sample flag is set else false.  
 
 | Param | Type | Description |
 | --- | --- | --- |
-| item | <code>string</code> \| [<code>Event</code>](#Event) \| <code>Metadata</code> | the item to get the sampling flag of |
+| item | <code>string</code> \| [<code>Event</code>](#Event) \| <code>addon.Event</code> | the item to get the sampling flag of |
```

Comment:
I should be consistent and choose one. It shouldn't be referred to as both. "addon" is the generic name for node addons; "bindings" is more of an internal name. so i'll use "addon" (even though the module name is "appoptics-bindings" - most people don't see that.)

internally i'm moving to using "aob" for "appoptics bindings" because it's an idiom in the implementation and should be both short and not cryptic. aob suggests appoptics-bindings pretty well.

on the "|" what is your IDE? if it's a big deal I can change them but i'm pretty sure no one will know what &#124; is off the top of their head whereas most unix-developers (is that redundant) expect \ means quote. what do you think?

Reply:
my IDE is IntelliJ. I'm not so worried about it looking correct here.
The question is: Where else besides of github is the page rendered and does it render correctly?

Reply:
actually, i just realized that this isn't going to get changed. the api.md is generated using jsdoc2md which converts the javascript code comments into markdown; changing that is not trivial, so until someone submits a bug...